devices.js

```diff
@@ -2394,7 +2394,13 @@ const devices = [
         fromZigbee: [fz.DTB190502A1_parse],
         toZigbee: [tz.DTB190502A1_LED],
     },
-
+    {
+        zigbeeModel: ['DTB-ED2004-012'],
+        model: 'DTB-ED2004-012',
+        vendor: 'databyte.ch',
+        description: 'Panda 1 - wall switch (https://databyte.ch/?post_type=portfolio&p=1818)',
+        extend: generic.light_onoff_brightness,
```

Comment:
Can this device also do dimming (brightness)? If not this should be changed to something like https://github.com/Koenkk/zigbee-herdsman-converters/blob/963eab430cccbc40457f29012e38b293368564f0/devices.js#L1008

Reply:
Yes, you are right. It can not do dimming. I have changed to fz.on_off. Thanks!